packages/fsweb/proxy/server.dev.js

```diff
@@ -0,0 +1,43 @@
+const exec = require('child_process').exec;
+const path = require('path');
+const Proxy = require('./proxy');
+
+const port = process.env.PORT || 3000;
+const hostname = process.env.HOST || 'localhost';
+const envPath = process.env.ENV_PATH || '../../env';
```

Comment:
This seems like it needs to be: (also on prod)
`const envPath = process.env.ENV_PATH || 'env/env';`

Reply:
@Cauldrath done